src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -1,99 +1,71 @@
 // This component will be modified according to the new wirefrime
 
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
 import {
-  FormGroup, ModalDialog, useToggle, Form,
+  FormGroup, ModalDialog, Form, ModalCloseButton,
 } from '@edx/paragon';
 import { Button } from 'react-paragon-topaz';
-import { handleInstructorsEnrollment } from 'features/Instructors/data/api';
 import { logError } from '@edx/frontend-platform/logging';
-import { fetchClassesData } from 'features/Instructors/data/thunks';
 
-const AddInstructors = () => {
-  const stateInstructors = useSelector((state) => state.instructors.classes);
+import { addInstructor } from 'features/Instructors/data/thunks';
+
+const AddInstructors = ({ isOpen, close }) => {
   const dispatch = useDispatch();
-  const [isOpen, open, close] = useToggle(false); // eslint-disable-line no-unused-vars
-  const [isNoUser, setIsNoUser] = useState(false);
-  const enrollmentData = new FormData();
+  const selectedInstitution = useSelector((state) => state.main.selectedInstitution);
 
-  const handleAddInstructors = async (e) => {
+  const handleAddInstructor = async (e) => {
     e.preventDefault();
     const form = e.target;
     const formData = new FormData(form);
     const formJson = Object.fromEntries(formData.entries());
     try {
-      enrollmentData.append('unique_student_identifier', formJson.instructorInfo);
-      enrollmentData.append('rolename', 'staff');
-      enrollmentData.append('action', 'allow');
-      const response = await handleInstructorsEnrollment(enrollmentData, formJson.ccxId);
-      if (response.data?.userDoesNotExist) {
-        setIsNoUser(true);
-      } else {
-        close();
-        setIsNoUser(false);
-      }
+      dispatch(addInstructor(selectedInstitution.id, formJson.instructorEmail));
+      close();
```

Comment:
We won't add confirmations at the end? @AuraAlba

Reply:
This is not part of the scope but I added the success message